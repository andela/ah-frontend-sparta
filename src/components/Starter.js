import React, { Component } from 'react';
import StarterContainer from './StarterContainer';

class Starter extends Component {
  render() {
    return (
      <div>
        <h1>Axios Prompt</h1>
        <StarterContainer />
      </div>
    );
  }
}
export default Starter;
