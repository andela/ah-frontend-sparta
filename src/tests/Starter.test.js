import React from 'react';
import renderer from 'react-test-renderer';
import Starter from '../components/Starter';

describe('Render dummy starter page from JSONPlaceholder', () => {
  it('should render without crashing', () => {
    const wrapper = renderer.create(<Starter />);
    expect(wrapper.toJSON()).toMatchSnapshot();
  });
});
