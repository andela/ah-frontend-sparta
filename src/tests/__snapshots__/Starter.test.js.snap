// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render dummy starter page from JSONPlaceholder should render without crashing 1`] = `
<div>
  <h1>
    Axios Prompt
  </h1>
  <div />
</div>
`;
